README.md

```diff
@@ -46,47 +36,29 @@ This project relies on a specific directory structure in order for it to reliabl
 > mkdir dams_ingest
 > mkdir dams_derivatives
 
-# Create development env file in `docker` directory
-> touch docker/.env.development
-
-# Start Docker from within `docker` directory
-> cd docker
-> docker-compose up
+# Start the application using Stack Car
+> cd hyrax
```

Comment:
cd stack_car

Reply:
I don't have to go into hyrax/stack_car. Is that default or did I edit my $path?

Reply:
My dev setup has stack_car folder where I do sc up. 
It can also be just cd /hyrax-->sc up or cd /hyrax/stack_car --> docker-compose up

Reply:
so like this?
```
# Start the application using Stack Car
> cd hyrax; sc up
# or
> cd hyrax/stack_car; docker-compose up
```

Reply:
yes.